nss-day-cohort-45/nutyshell-talking-to-strangers-1
# Public messages

# Description

This branch creates a form for new messages, converts to HTML, saves message to database, displays Messages on the DOM

Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
- serve json (need at least one message in the database)
- serve webpage
- check the messages are displayed (message text, date, and username)
- check that message put into the form are displayed in the Chat History 
- check that messages are deleted

# Checklist:
- [ ] My changes generate no new warnings or errors **500 (internal Server Error)**
